src/Widgets/DataSearchWidget/DataSearchWidgetComponent.tsx

```diff
@@ -43,7 +44,11 @@ provideComponent(DataSearchWidget, ({ widget }) => {
         defaultValue={search}
       />
       <span className="input-group-btn">
-        <button className={`btn ${buttonColor}`} type="submit">
+        <button
+          className={`btn ${buttonColor}`}
+          type="submit"
+          aria-label={localizeSearchInputLabel()}
```

Comment:
* [ ] FYI looking at the current values for `placeholder` (typically "Search for X") we could repurpose these and use them for both the placeholder and the button label. If we do so, it should be made clear in the editing config though.